src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -155,32 +164,136 @@ export function EmbeddedPositionSearchPanel({
         ? <WrapperComponent {...additionalProps}>{internals}</WrapperComponent>
         : internals;
 
+    function SearchSummary({title, searchType, coords, radius, fileName, rows, showCoords, showRadius, showFileInfo, sx}) {
+        return (
+            <Stack sx={sx}>
+                {title && <><Typography color={'warning'} level='title'>{title}</Typography></>}
+                <Typography color={'neutral'} level='body-md'>
+                    <Typography component='span' color={'primary'}>Search Type</Typography>: {searchType}
+                    {showRadius && searchType === 'Cone' && <>
+                        , <Typography component='span' color={'primary'}>Search Radius</Typography>: {radius}
+                    </>}
+                    {showCoords && searchType !== 'Multi-Object' && coords &&
+                        <>
+                            , <Typography component='span' color={'primary'}>Coordinates</Typography>: {coords}
+                        </>
+                    }
+                    {showFileInfo && searchType === 'Multi-Object' && fileName && rows &&
+                        <>
+                            , <Typography component='span' color={'primary'}>File Name</Typography>: {fileName}
+                            , <Typography component='span' color={'primary'}>Rows</Typography>: {rows}
+                        </>
+                    }
+                </Typography>
+            </Stack>
+        );
+    }
+
+    const Header = function(isOpen) {
+        const {groupKey} = useContext(FieldGroupCtx);
+        const reqObj = getFieldGroupResults(groupKey,true);
+
+        const searchType = doGetConeAreaOp() === CONE_CHOICE_KEY ? 'Cone' : (doGetConeAreaOp() === POLY_CHOICE_KEY  ? 'Polygon' : 'Multi-Object');
+        const coords = searchType === 'Cone' ? (reqObj?.UserTargetWorldPt || reqObj?.circleTarget) : (reqObj?.Polygon || reqObj?.['POS-polygon']);
+        const radius = reqObj?.radius || reqObj?.circleSize;
+
+        //in case of Multi-Object, get the fileName & rows
+        const fileName = searchType === 'Multi-Object' ? reqObj?.uploadInfo?.fileName : undefined;
+        const rows = searchType === 'Multi-Object' ? reqObj?.uploadInfo?.totalRows : undefined;
+
+        const defSlotProps = {
+            searchSummary: {
+                title: undefined,
+                searchType, coords, radius, fileName, rows,
+                showCoords: false,
+                showRadius: false,
+                showFileInfo: false,
+                sx:{}
+            },
```

Comment:
About this, you're right about the first part. I need not add `searchType, coords, radius, fileName, rows` to searchSummary/defSlotProps as these are not meant to be overridden. So I'll just pass these directly to `SearchSummary`. 

As for the second part, @robyww wanted the Search Summary to be customizable by whoever's calling it (for example: see how I'm calling it for Euclid here: https://github.com/IPAC-SW/irsa-ife/pull/323/files). DCE may want to call it differently, and for instance, may not want to show `radius` even when it's available. In fact, even though `title` does take up extra space, it can optionally be left empty as well. 

That was the idea behind keeping the Search Summary customizable by any component calling `EmbeddedPositionSearchPanel`... But I am open to better ideas (or even a better default). Right now the default is to just show the `Search Type` (`Cone/Polygon/Multi-object`) and nothing else. And I can get rid of the `title` option altogether if it seems like there would be no good use case for it.